HANDBOOK.md

```diff
@@ -65,7 +61,6 @@ SDR was built to accomplish the task of deploying and retrieving metadata. There
 
 🧽 **Refactor/Clean** - Nice to tidy up. Not crucial, they’re just nit picky things that the author, Bryan, dislikes. If someone feels similarly about cleaning up then go forth and conquer
 
-
 ## Metadata Registry
 
 **Module:** https://github.com/forcedotcom/source-deploy-retrieve/tree/develop/src/registry
```

Comment:
I know this has nothing to do with your changes, but there are a lot of references to a `develop` branch that doesn't exist. Could you update these to `main`?